app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -413,6 +413,9 @@ def vms
       # will take care of that.
       if s.attributes.fetch("os-extended-volumes:volumes_attached", []).length > 0
         s.volume_attachments.each do |attachment|
+          # Skip Volume mounts without mount point
+          next unless attachment['device']
```

Comment:
will `attachment['device']` be nil if the mountpoint is missing, or can it contain the empty string? Wondering if it might be better to test with `.blank?

Reply:
Good point, updated, thanks :+1: